src/Simplex/Chat/Messages.hs

```diff
@@ -524,13 +537,42 @@ dbJsonCIContent = \case
   CIRcvMsgContent mc -> DBJCIRcvMsgContent mc
   CISndDeleted cidm -> DBJCISndDeleted cidm
   CIRcvDeleted cidm -> DBJCIRcvDeleted cidm
+  CISndCall status duration -> DBJCISndCall {status, duration}
+  CIRcvCall status duration -> DBJCIRcvCall {status, duration}
 
 aciContentDBJSON :: DBJSONCIContent -> ACIContent
 aciContentDBJSON = \case
   DBJCISndMsgContent mc -> ACIContent SMDSnd $ CISndMsgContent mc
   DBJCIRcvMsgContent mc -> ACIContent SMDRcv $ CIRcvMsgContent mc
   DBJCISndDeleted cidm -> ACIContent SMDSnd $ CISndDeleted cidm
   DBJCIRcvDeleted cidm -> ACIContent SMDRcv $ CIRcvDeleted cidm
+  DBJCISndCall {status, duration} -> ACIContent SMDSnd $ CISndCall status duration
+  DBJCIRcvCall {status, duration} -> ACIContent SMDRcv $ CIRcvCall status duration
+
+data CICallStatus
+  = CISCallPending
+  | CISCallMissed
```

Comment:
How is Missed status achieved in state machine? Is it when the call wasn't successfully negotiated? Would it be the same when the call couldn't be negotiated due to an error?

Reply:
When there was no response and it ended. Probably there should be one more status for the error